go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/bitrise-steplib/steps-go-test
+
+go 1.23.2
```

Comment:
We are running this step on the Xcode 16.2 stack which does not have this Go version:
```
- 1.21.13
- 1.21
- 1.22.7
- 1.22 (default)
- 1
```
I think this might cause some problems.